### ISSUE TITLE
Json parsing throw warnings on arrays expecting strings

### ISSUE THREAD

**Issue by thenanox** [mulesoft/api-workbench#139]
Since last version, exampled defined in json throw warning when an array is defined saying String is expected:

First we define the data type. We can check the array identityDocuments defined:
![string1](https://cloud.githubusercontent.com/assets/4981277/13844283/3063c044-ec39-11e5-97b5-9e991702bf2c.png)

In our RAML now the json used check that property and says string is expected:
![string2](https://cloud.githubusercontent.com/assets/4981277/13844311/5522856e-ec39-11e5-82cd-26f3e1364a50.png)

**Comment by blakeembrey** [comment]
@thenanox I haven't confirmed, but you're writing YAML there. If you do `content: |` it'll switch to writing it as text in YAML. Is that your issue?

Including the text instead of screenshots can help people reproduce and debug your issue too.

**Comment by thenanox** [comment]
@blakeembrey what i'm trying is to include a json as an example of answer. So the first screen shows the dataType this resource is using. The second screen is an example created in json format that fit in that data type

This is the code that generates this error:

```
#%RAML 1.0
title: user
version: V0
baseUri: https://www.domain.com/user/v0

types:
  user:
    properties:
      data:
        properties:
          userId: string
          firstName: string
          lastName: string
          lastAccessDate: string
          email: string
          phoneNumber: string
          sex: string
          identityDocuments:
            type: array
            items:
              properties:
                documentNumber: string
                documentType:
                  properties:
                    id: string
                    name: string
            minItems: 1
          address:
            properties:
              country: string
              region: string
              city: string
              address: string
              zipCode: string
          birthDate: string
          isEmployee: boolean
          avatar:
            properties:
              id: string
              name: string
              url: string

/users/{userId}:
  description:  |
    Service for retrieving contextual information related to the user logged to the API.
  uriParameters:
    userId:
      description: "User identifier"
      type: string
      example: E123456

  get:
    description:  |
      Service for retrieving contextual information related to the user logged to the API. The result is a diverse set of user data such as name, last access date, etc.

      This information is very useful to consumers in order to know in detail the interests, preferences of the user. So that consumers can decide which services should be invoked (or not) to provide better service to the user.

    responses:
      200:
        description: !include v1/user/docs/usersContextData_200.md
        body:
          application/json:
            type: user
            examples:
              - content:
                  {
                    "data": {
                      "userId": "E123456",
                      "firstName": "Amy",
                      "lastName": "William Moure",
                      "lastAccessDate": "2015-10-06T09:00:33Z",
                      "email": "amy.william@gmail.com",
                      "phoneNumber": "612345678",
                      "sex": "FEMALE",
                      "identityDocuments": [
                        {
                          "documentType": {
                              "id": "PASSPORT",
                              "name": "Passport"
                            },
                          "documentNumber": "00000000-A"
                        }
                      ],
                      "address": {
                        "country": "Spain",
                        "region": "Comunidad de Madrid",
                        "city": "Madrid",
                        "address": "Avenida lucio",
                        "zipCode": "28029"
                      },
                      "birthDate": "1980-03-02",
                      "isEmployee": true,
                      "avatar": {
                        "id": "12345678",
                        "name": "Profile picture",
                        "url": "https://.../img/FEMALE.jpg"
                      }
                    }
                  }
```

**Comment by blakeembrey** [comment]
@ddenisenko Is this the same issue with annotations where `array` types no longer understand the `items` property?

**Comment by VasiliyLysokobylko** [comment]
To fix this error, please remove `-` symbol in the string `- content:`.

**Comment by ddenisenko** [comment]
Works with map syntax in `code` branch, which should be published in 0.8.30.
